components/Header.js

```diff
@@ -5,6 +5,8 @@ import Login from "./Login"
 const HeaderWrapper = styled.article`display: flex;
 justify-content: space-between;
 align-items: center;
+position: fixed; z-index: 1;
+padding-top: 0; margin-top: 0;
```

Comment:
something seems to be broken with your code formatter. Please check that it automatically formats your styled components and add a linebreak after each property.